frontend/src/store/modules/interactionPartners.js

```diff
@@ -15,7 +15,10 @@ const data = {
 const getters = {
   component: state => state.interactionPartners.component || {},
   reactions: state => state.interactionPartners.reactions || [],
-  title: (state, _getters) => (_getters.component.type === 'metabolite' ? chemicalName(_getters.component.name) : _getters.component.name),
+  title: (state, _getters) => (_getters.component.type === 'metabolite'
+    ? chemicalName(_getters.component.name || _getters.component.id)
```

Comment:
IDs should not be interpreted as `chemicalName`.

```suggestion
    ? chemicalName(_getters.component.name)
```

Reply:
Nice! I was too sloppy with that. The metabolites didn't seem to have a problem anyways but do you think we should still try to account for the name possible being null? In that case it would be: 
```javascript
_getters.component.name ? chemicalName(_getters.component.name) : _getters.component.id
```

Reply:
Would this be applied to `'gene'` as well then?
`chemicalName` checks for null btw.

Reply:
> Would this be applied to `'gene'` as well then?

Can you elaborate on this? Not sure what you mean.

___

> `chemicalName` checks for null btw.

While it checks for null it doesn't return anything usable, so it would show "Interaction Partners for" if the name is null.

Reply:
> IDs should not be interpreted as `chemicalName`.

I agree.

Reply:
> > Would this be applied to `'gene'` as well then?
> 
> Can you elaborate on this? Not sure what you mean.

The Interaction Partners tool works for both genes and metabolites, and it is only the metabolites that should be passed to the `chemicalName` function. At a quick glance, it looked like the genes would also go through that parsing - it might not lead to any changes, but it should still be avoided.

Reply:
Nevermind the comment above, I see know it shouldn't be a problem!